### PR TITLE
added in the content for your estimates and the changes in your income

### PR DESCRIPTION
**Pull request by etheya**
Have added the content in that was still missing from the live version and included the work for the ticket CCALC-4082. Dont think this needs any logic around as it should be relevant for every result and you wouldnt get here without getting something as GREEN.

I havent added the code to the tests so if this needs to be added in then someone will need to check this branch out first.

### CODE REVIEW

**Comment by etheya** [comment]
Re ran tests from fresh, all passed

File: app/uk/gov/hmrc/childcarecalculatorfrontend/views/result.scala.html [hunk 1]
```diff
@@ -79,6 +80,7 @@ <h2 class="heading-medium">@messages("results.two.years.old.guidance.title")</h2
 
         @changesToCircumstancesContent
```
Comment (versatil56) [inline app/uk/gov/hmrc/childcarecalculatorfrontend/views/result.scala.html]: We need logic around here to only show this when we have at least 1 green. While the assumption that we would never hit /results without at least 1 GREEN is correct, a user could easily scenario play and get browse directly to /result with an all negative scenario, at which point is would be wrong.

File: app/uk/gov/hmrc/childcarecalculatorfrontend/views/result.scala.html [hunk 2]
```diff
@@ -33,8 +33,9 @@
 }
 
 @changesToCircumstancesContent = {
```
Comment (versatil56) [inline app/uk/gov/hmrc/childcarecalculatorfrontend/views/result.scala.html]: The name for this variable is not meaningful anymore, can we rename to something better. Is it needed?

**Comment by sara202** [comment]
@versatil56 changes have been made as requested

**Pull request merged**